_episodes/00-short-introduction-to-Python.md

```diff
@@ -24,7 +24,6 @@ Python's main advantages:
 * Available on all platforms
 * "Batteries Included" philosophy - libraries for common tasks available in
   standard installation
-* Supports multiple programming paradigms
```

Comment:
If the lesson is for novices, then this point is too advanced and too likely to raise unnecessary questions/confusion.